### DISCUSSION TITLE
Where are the v2 docs?

### DISCUSSION THREAD

**Discussion by givihuda** [ColorlibHQ/AdminLTE#4741] [Q&A]
The docs go to version 3: https://adminlte.io/docs/3.2/index.html

However, nothing selectable for version 2.

I'm using version 2.3.2 (not my choice, sorry), and just trying to learn how it works so far without going down too many irrelevant paths.

**Comment by dfsmania** [comment]
Just a simple google search gives me the next url:

https://adminlte.io/themes/AdminLTE/documentation/

**Reply by givihuda** [reply]
OK so your Google search found v2 (whatever words you used), but since when should docs rely on Google to find different versions of them?

Also, that URL doesn't mention the version number... why? v3 has the version number in it.

The docs link in the repo's current master ReadMe go to https://adminlte.io/docs/3.2, same with the link in the v3 branch.

The docs link in the v2 branch ReadMe point to https://adminlte.io/docs which redirects to https://adminlte.io/docs/3.2 😆 and the drop-down from there only has v3.x.

So what you're missing is that for people who need to view the v2 docs, the v3 doc page should include a link or drop-down selection to go to v2, not have them rely on Google or some other search engine which may not find them the right thing.

**Reply by dfsmania** [reply]
I was just trying to help you found the `v2` docs. However, I think that version is no longer maintained, but your suggestion is nice. Maybe the package maintainers take it into consideration.
